### PR TITLE
Use a different InternalsVisibleTo notation in the project files.

### PR DESCRIPTION
**Pull request by dlemstra**

### CODE REVIEW

**Comment by dlemstra** [comment]
@dtivel I rebased the PR to make sure it's now up to date with the main branch. Would be nice if this gets merged soon so I can update my 703 PR.

File: src/Sign.Cli/Sign.Cli.csproj [hunk 1]
```diff
@@ -24,12 +24,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
-      <_Parameter1>Sign.Cli.Test,PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9</_Parameter1>
-    </AssemblyAttribute>
-    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
-      <_Parameter1>DynamicProxyGenAssembly2,PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7</_Parameter1>
-    </AssemblyAttribute>
+    <InternalsVisibleTo Include="Sign.Cli.Test" />
```
Comment (dtivel) [inline src/Sign.Cli/Sign.Cli.csproj:27]: @dlemstra, can you elaborate on removing public key information for projects in this solution?

Comment (clairernovotny) [reply src/Sign.Cli/Sign.Cli.csproj:27]: This is a new MSBuild SDK feature that puts the correct value into dependent assemblies.

**Review by clairernovotny — DISMISSED** [review]
Can the `DynamicProxyGenAssembly2 information be added to a directory import so it doesn't need to be repeated?

**Comment by dlemstra** [comment]
Yeah it would be a good idea to move that there @clairernovotny. I have re-based the PR and changed my commit to include your suggested change.

**Review by dtivel — APPROVED** [review]

**Pull request merged**